### PR TITLE
Better, added VL.

### PR DESCRIPTION
**Pull request by xZeroMCPE**

### CODE REVIEW

**Comment by hugmanrique** [comment]
Please, don't create additional PR and commit changes to your previous one instead. [Here's how](https://help.github.com/articles/adding-an-existing-project-to-github-using-the-command-line/)

**Comment by xZeroMCPE** [comment]
@hugmanrique I closed the previous one.

File: config.json [hunk 1]
```diff
@@ -11,13 +11,15 @@
...
 		"/favicons/pigraid.png": "assets/images/favicons/pigraid.png"
+		"/favicons/vastlands.png": "assets/images/favicons/vastlands.png"
 	},
 	"faviconOverride": {
 		"Mineplex": "/favicons/mineplex.png",
 		"Lifeboat": "/favicons/lifeboat.png",
 		"InPvP": "/favicons/inpvp.png",
 		"CookieBuild": "/favicons/cookie_build.png",
 		"PigRaid": "/favicons/pigraid.png"
+		"VastLands": "/favicons/vastlands.png"
```
Comment (dktapps) [inline config.json]: seems something is missing here 😉

Comment (Chewsterchew) [reply config.json]: good job you forgot a comma

**Pull request closed without merging**